pkg/agent/route/route_windows.go

```diff
@@ -88,6 +96,9 @@ func (c *Client) Reconcile(podCIDRs []string) error {
 			c.hostRoutes.Store(dst, rt)
 			continue
 		}
+		if c.nodeConfig.UplinkNetConfig != nil && rt.LinkIndex != c.nodeConfig.UplinkNetConfig.Index {
```

Comment:
Add some comments about when this can happen?

Reply:
Updated.